pkg/logql/log/label_filter_test.go

```diff
@@ -26,10 +26,10 @@ func TestBinary_Filter(t *testing.T) {
 			labels.Labels{{Name: "foo", Value: "5"}, {Name: "bar", Value: "1s"}},
 		},
 		{
-			NewAndLabelFilter(NewNumericLabelFilter(LabelFilterEqual, "foo", 5), NewBytesLabelFilter(LabelFilterEqual, "bar", 42)),
-			labels.Labels{{Name: "foo", Value: "5"}, {Name: "bar", Value: "42B"}},
+			NewAndLabelFilter(NewNumericLabelFilter(LabelFilterEqual, "foo", 5), NewBytesLabelFilter(LabelFilterEqual, "bar", 42000)),
+			labels.Labels{{Name: "foo", Value: "5"}, {Name: "bar", Value: "42kB"}},
```

Comment:
I think it's time for building a big table test focusing on the byte filter. You can take the skeleton of this one and create a new tests with only bytes tests. this way you can add many of them not need to use AndFilter.

Reply:
Good idea. I actually thought we should have a matrix for this test.